pkg/placement/apiwatch-map-provider.go

```diff
@@ -92,10 +88,25 @@ func (awp *apiWatchProvider) AddReceivers(clusterName logicalcluster.Name,
 			groupReceivers:    MappingReceiverHolderFork[string /*group name*/, APIGroupInfo]{},
 			resourceReceivers: MappingReceiverHolderFork[metav1.GroupResource, ResourceDetails]{},
 		}
-		discoveryScopedClient := awp.discoveryClusterClient.Cluster(clusterName.Path())
-		crdInformer := awp.crdClusterPreInformer.Cluster(clusterName).Informer()
-		bindingInformer := awp.bindingClusterPreInformer.Cluster(clusterName).Informer()
-		wpc.informer, wpc.lister, _ = apiwatch.NewAPIResourceInformer(ctx, clusterName.String(), discoveryScopedClient, false, crdInformer, bindingInformer)
+
+		config, err := awp.spaceclient.ConfigForSpace(clusterName, awp.spaceProviderNs)
```

Comment:
Set the UserAgent.

Reply:
Sorry for unsolicited comment. Just curious, what is UserAgent used for? @MikeSpreitzer

Reply:
It appears in the log messages of the apiserver.